package.json

```diff
@@ -225,22 +225,30 @@
     "cover-integration": "cross-env COVERAGE=true nyc --reporter=lcovonly yarn mocha \"test/integration/**/*.test.[tj]s\" && node -e \"require('fs').renameSync('coverage/lcov.info', 'coverage/integration.info')\"",
     "cover-unit": "cross-env COVERAGE=true nyc --reporter=lcovonly yarn mocha \"test/unit/**/*.test.[tj]s\" && node -e \"require('fs').renameSync('coverage/lcov.info', 'coverage/unit.info')\"",
     "merge-coverage": "lcov-result-merger \"coverage/*.info\" \"coverage/lcov.info\"",
-    "----------------------------------------- local test dbs ------------------------------------------": "",
+    "----------------------------------------- bulk tests ------------------------------------------": "",
+    "start-all": "grep 'bash dev' package.json | grep start | grep -vP \"(all)\" | cut -d'\"' -f4 | sort | xargs -n1 | grep dev | xargs -n1 bash",
```

Comment:
A `start-all` script doesn't really make sense as ports are only unique per database type, not per database version. You can't start postgres & postgres-10 at the same time.

A `start-all-minimum` (starts all minimum supported versions) and `start-all-maximum` (starts all maximum supported versions) would make sense, no port collision there, but right now we have commands such as `start-mysql` and `start-mysql-8` instead of `start-mysql-minimum` and `start-mysql-maximum`